test/language/statements/class/subclass/builtin-objects/Date/regular-subclassing.js

```diff
@@ -20,18 +20,18 @@ info: >
 
 class D extends Date {}
 
-var d1 = new D(1859, '10', 24);
-assert.sameValue(d1.getUTCFullYear(), 1859);
-assert.sameValue(d1.getUTCMonth(), 10);
-assert.sameValue(d1.getUTCDate(), 24);
+var d1 = new D(1859, '10', 24, 11);
+assert.sameValue(d1.getFullYear(), 1859);
+assert.sameValue(d1.getMonth(), 10);
+assert.sameValue(d1.getDate(), 24);
 
 var d2 = new D(-3474558000000);
-assert.sameValue(d2.getUTCFullYear(), 1859);
-assert.sameValue(d2.getUTCMonth(), 10);
-assert.sameValue(d2.getUTCDate(), 24);
+assert.sameValue(d2.getFullYear(), 1859);
+assert.sameValue(d2.getMonth(), 10);
+assert.sameValue(d2.getDate(), 24);
 
 var d3 = new D();
 var d4 = new Date();
-assert.sameValue(d3.getUTCFullYear(), d4.getUTCFullYear());
-assert.sameValue(d3.getUTCMonth(), d4.getUTCMonth());
-assert.sameValue(d3.getUTCDate(), d4.getUTCDate());
+assert.sameValue(d3.getFullYear(), d4.getFullYear());
+assert.sameValue(d3.getMonth(), d4.getMonth());
+assert.sameValue(d3.getDate(), d4.getDate());
```

Comment:
If we compare the time values instead, wouldn't that be enough to verify that the Date internals are initialized correctly? Since we already use `getDate` etc. together with other variants of the constructor, we already cover how the time is parsed.

This change isn't needed for this PR by the way. It's just something to consider for the future.